cli/module_registry.go

```diff
@@ -252,10 +252,17 @@ func UploadModuleAction(c *cli.Context) error {
 			return errors.Errorf("module name %q was supplied on the command line but the meta.json has a module ID of %q", nameArg,
 				moduleID.name)
 		}
-	}
-	moduleID, err = validateModuleID(client, moduleID.String(), publicNamespaceArg, orgIDArg)
```

Comment:
I moved this inside of the "meta.json file exists" if block. I think it makes sense because its specifically checking whether args of the cli match the manifest file. Doesn't need to be done if the manifest doesn't exist.